src/ui/toolbar/components/toolbar-button.vue

```diff
@@ -36,7 +36,7 @@ export default {
     },
     methods: {
         onClick(event) {
-            if (this.options.dialog) {
+            if ((this.options.isEditing === undefined || this.options.isEditing) && this.options.dialog) {
```

Comment:
i see `(this.options.isEditing === undefined) || this.options.isEditing)` several times. i know it'd be extra code but i wonder if this should be in a computed with a descriptive name so that someone doesn't come along and mistakenly stomp over this thinking it can be a simple truthy check.

Reply:
OK.

Reply:
Method would work fine if used for pure js logic. The computed would allow vue to cache the result if this is to be used in templates.